9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
@mnagy yeah, that seems like a good idea -- there can be one or more useful examples in the example/extending/ directory with necessary comments. One example might show how to tune more configuration options, another how to run more SQL commands during initialization and another how to install some plugin... (those seem to me like the most common reasons why somebody would extend the image)
@pkajaba @praiskup what do you think?